### PR TITLE
[TW-340] Update changelog docs

### PR DESCRIPTION
**Pull request by claireriley**

### CODE REVIEW

File: src/pages/docs/collaborating-in-postman/using-workspaces/changelog-and-restoring-collections.md [hunk 1]
```diff
@@ -15,26 +15,28 @@ warning: false
 
 > **[Changelog is available on Postman Basic, Professional, and Enterprise plans.](https://www.postman.com/pricing/)**
 
-Your Postman collections display a changelog for reviewing create, update, and delete events. You can use the changelog to keep track of any updates you and your collaborators make to your private and team collections. The changelog also lets you rollback a collection and restore it to any previous point in time. Postman also tracks activity within teams and accounts.
+Each Postman Collection has a changelog that covers create, update, and delete events. You can use the changelog to keep track of updates you and your collaborators make to your private and team collections. The changelog also lets you rollback a collection and restore it to an earlier point in time.
```
Comment (kartikjain26) [inline src/pages/docs/collaborating-in-postman/using-workspaces/changelog-and-restoring-collections.md:18]: We can include transfer also here.

Comment (kartikjain26) [reply src/pages/docs/collaborating-in-postman/using-workspaces/changelog-and-restoring-collections.md:18]: Can change this `you and your collaborators` to `you and other collaborators`

File: src/pages/docs/collaborating-in-postman/using-workspaces/changelog-and-restoring-collections.md [hunk 2]
```diff
@@ -78,23 +80,21 @@ You can review the activity for a team with a Postman Basic, Professional, or En
 
 ## Viewing user activity
 
-You can review the activity for your own account in [Postman](https://go.postman.co/). Select your avatar in the top right, and select __Profile__ — the __Activity__ tab will list your events.
-
-## Restoring collections and folders
-
-With a Postman Basic, Professional, or Enterprise account, you can use the changelog to restore a collection to a previous point in time. Select __Restore__ under a change to revert the collection to the point immediately _after_ the change was applied. The changelog will update and Postman will display a confirmation message indicating the time the collection has been restored to.
+You can review the activity for your own account in [Postman](https://go.postman.co/). Select your avatar in the top right, then select __Profile__. The __Activity__ tab will list your events.
```
Comment (kartikjain26) [inline src/pages/docs/collaborating-in-postman/using-workspaces/changelog-and-restoring-collections.md:83]: do we need this in changelog ?

**Review by kartikjain26 — CHANGES_REQUESTED** [review]

**Review by HeatherStenson — APPROVED** [review]
Looks good! 🍊

**Review by jkonrath-postman — APPROVED** [review]
Looks good to me!

File: src/pages/docs/collaborating-in-postman/using-workspaces/changelog-and-restoring-collections.md [hunk 3]
```diff
@@ -15,26 +15,28 @@ warning: false
 
 > **[Changelog is available on Postman Basic, Professional, and Enterprise plans.](https://www.postman.com/pricing/)**
 
-Your Postman collections display a changelog for reviewing create, update, and delete events. You can use the changelog to keep track of any updates you and your collaborators make to your private and team collections. The changelog also lets you rollback a collection and restore it to any previous point in time. Postman also tracks activity within teams and accounts.
+Each Postman Collection has a changelog that covers create, update, and delete events. You can use the changelog to keep track of updates you and your collaborators make to your private and team collections. The changelog also lets you rollback a collection and restore it to an earlier point in time.
 
 ## Contents
 
 * Viewing changes to a [collection](#viewing-the-collection-changelog), [workspace](#viewing-workspace-activity), [team](#viewing-team-activity), or [user account](#viewing-user-activity).
```
Comment (mark-anthony-postman) [inline src/pages/docs/collaborating-in-postman/using-workspaces/changelog-and-restoring-collections.md:22]:
```suggestion
* Viewing changes to a [collection](#viewing-the-collection-changelog), [workspace](#viewing-workspace-activity), [team](#viewing-team-activity), or [user account](#viewing-user-activity)
```

File: src/pages/docs/collaborating-in-postman/using-workspaces/changelog-and-restoring-collections.md [hunk 4]
```diff
@@ -78,23 +80,21 @@ You can review the activity for a team with a Postman Basic, Professional, or En
...
-If you accidentally delete a folder, you can recover it by selecting __Restore__.
+You can use the [changelog](#viewing-the-collection-changelog) to restore a collection to an earlier point in time.
 
-<img alt="Recover folder" src="https://assets.postman.com/postman-docs/restore-folder-v8.jpg" width="400px"/>
+To restore a collection from the changelog:
 
-You may see a warning on any request tabs you have open that are affected by reverting the collection, for example indicating a conflict or deletion.
+1. Select a session to expand it.
+2. Select the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> next to the change you'd like to restore your collection to, then select **Restore to this change**.
+    <img alt="Restore from changelog" src="https://assets.postman.com/postman-docs/changelog-restore-v9.16.jpg" width="350px"/>
```
Comment (mark-anthony-postman) [inline src/pages/docs/collaborating-in-postman/using-workspaces/changelog-and-restoring-collections.md:93]: Add a blank line before the image

File: src/pages/docs/sending-requests/intro-to-collections.md [hunk 5]
```diff
@@ -239,10 +239,12 @@ To add a comment at the collection level:
 
 To revert your collection to a previous state:
 
-1. Select the changelog icon <img alt="Changelog icon" src="https://assets.postman.com/postman-docs/icon-changelog-v9.jpg#icon" width="18px"> from the right of the collection overview.
-1. Select **Restore** for the state to which you want to revert the collection.
+1. Select the changelog icon <img alt="Changelog icon" src="https://assets.postman.com/postman-docs/icon-changelog-v9.jpg#icon" width="18px"> on the right.
+2. Select a session to expand it.
+3. Select the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> next to the change you'd like to restore your collection to, then select **Restore to this change**.
+    <img alt="Restore from changelog" src="https://assets.postman.com/postman-docs/changelog-restore-v9.16.jpg" width="350px"/>
```
Comment (mark-anthony-postman) [inline src/pages/docs/sending-requests/intro-to-collections.md:245]: Add a blank line before the image

**Review by mark-anthony-postman — APPROVED** [review]
LGTM! 🌟  Just a couple small comments

File: src/pages/docs/collaborating-in-postman/using-workspaces/changelog-and-restoring-collections.md (hunk 2 above)
Comment (claireriley) [reply src/pages/docs/collaborating-in-postman/using-workspaces/changelog-and-restoring-collections.md:83]: Looks like this is outdated, removed it

**Review by ankitjaggi — APPROVED** [review]

**Comment by ankitjaggi** [comment]
@claireriley We can merge this and release the new docs now.

**Pull request merged**